### PR TITLE
Added Documentation for installation FW Toolchain software

### PR DESCRIPTION
**Pull request by AlexAntn**
This PR introduces documentation for installing software (Keil, MPLAB) for the FW toolchain.

The pages should appear under `icub-firmware` section, in `fw_toolchain`.

### CODE REVIEW

**Review by pattacini — CHANGES_REQUESTED** [review]
Thanks @AlexAntn 👍🏻 

CI is failing w/:
```
WARNING -  A relative path to 'icub_firmware/fw_toolchain/installing_kiel.md' is included in the 'nav' configuration, which is not found in the documentation files 
WARNING -  Documentation file 'icub_tendons/waist_legs.md' contains a link to 'icub_tendons/img\waist\Waist_cable_11.png' which is not found in the documentation files. 
```

Could you deal with that?

Also, it's not `Kiel` but `Keil` 😄
Can you fix the typo, including the file names?

File: mkdocs.yml [hunk 1]
```diff
@@ -153,6 +153,9 @@ nav:
         - How to use two real iCub robots on the same network: icub_setup_multiple_robots/two_robots_real.md
       - iCub Firmware:
         - Summary: icub_firmware/index.md
+        - Firmware Toolchain Documentation: 
+          - Installing Kiel: icub_firmware/fw_toolchain/installing_keil.md
```
Comment (pattacini) [inline mkdocs.yml:157]:
```suggestion
          - Installing Keil: icub_firmware/fw_toolchain/installing_keil.md
```

Comment (AlexAntn) [reply mkdocs.yml]: Ok, now there should be no kiels anymore (I fixed the one on the path, didn't notice the title...)

File: docs/icub_firmware/fw_toolchain/installing_mplab.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+![image](img/MPLAB_1.png)
+
+The 2FOC device is the **dsPIC33FJ128MC802**.
+- Then click on Packs - dsPIC33F-GP-...   You should see a popup window with the project properties
+
+![image](img/MPLAB_2.png)
+
+just click "OK".
+- Clean and Build the project as shown below:
+<img width="1751" alt="image" src="img/MPLAB_3.png">
```
Comment (pattacini) [inline docs/icub_firmware/fw_toolchain/installing_mplab.md:88]:
```suggestion
just click "OK".

- Clean and Build the project as shown below:

![image](img/MPLAB_3.png)
```

**Review by pattacini — APPROVED** [review]
Nicely done! I've just checked the rendering on your fork and it looks pretty good!

**Pull request merged**